.eslintrc.json

```diff
@@ -26,7 +26,8 @@
         "get_locale_string": false,
         "__dirname": false,
         "__filename": false,
-        "__meta": false
+        "__meta": false,
+        "capitalize": false
```

Comment:
do we need `"clamp": false` here as well since that was added?

Reply:
yes, good catch